MovieTime/MovieTime/View/Movies/MTMovieDetailViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  MTMovieDetailViewController.swift
```

Comment:
currently the user can only read content from the movie details page. As a user or a product owner, what do you suggest as the next features for this page?